tests/test_telemetry.py

```diff
@@ -369,3 +370,26 @@ def test_app_dependencies_loaded(self):
         for dependency, seen in seen_loaded_dependencies.items():
             if not seen:
                 raise Exception(dependency + " not recieved in app-dependencies-loaded message")
+
+    def setup_message_batch_event_order(self):
+        weblog.get("/load_dependency")
+        weblog.get("/enable_integration")
+        weblog.get("/enable_product")
+
+    @missing_feature(
```

Comment:
If I understand correctly, this feature is not implemented anywhere.

The fact it's passing on other weblog is due to the fact that the entire test class is skipped for those test.

I think it will be cleaner to use a dedicated test class for this, let say `Test_ForceBatchingEnable`, WDYT ?

Reply:
Creating a seperate test Test_ForceBatchingEnable would be a good Idea. It wont hinder with other system test too

Reply:
In which file do we mention this that for this scenario run this test file?